src/admin/components/Alerts/DelayedJobsProgressAlert.tsx

```diff
@@ -12,47 +11,16 @@ type DelayedJobsProgressAlertProps = {
 };
 
 const DelayedJobsProgressAlert: FC<DelayedJobsProgressAlertProps> = ({ show, title, setIsLoadingDelayedJob }) => {
```

Comment:
I updated this component, but when I went to look into how to test it I found that it's completely unused. Are there plans to use it in the future, or is this dead code? If it's dead, I think we should go ahead and remove it in this PR. It's also the only place that ever calls `JobSlice.abortDelayedJob`, so if this isn't going to be used, we could rip out the delayed job abort code entirely.